package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Service Worker Toolbox provides some simple helpers for use in creating your own service workers.",
   "license": "Apache-2.0",
   "scripts": {
-    "publish-release": "./node_modules/sw-testing-helpers/project/publish-release.sh",
+    "publish-release": "./node_modules/npm-publish-scripts/publish-release.sh",
```

Comment:
Okay, that's merged. But you'll need to be the one to publish a new release of `npm-publish-scripts` to npm.